src/reanimated2/mutables.ts

```diff
@@ -1,23 +1,21 @@
 'use strict';
-import NativeReanimatedModule from './NativeReanimated';
 import { shouldBeUseWeb } from './PlatformChecker';
-import type { ShareableSyncDataHolderRef, Mutable } from './commonTypes';
-import {
-  makeShareableCloneOnUIRecursive,
-  makeShareableCloneRecursive,
-} from './shareables';
+import type { Mutable } from './commonTypes';
+import { makeShareableCloneRecursive } from './shareables';
 import { shareableMappingCache } from './shareableMappingCache';
-import { runOnUI } from './threads';
+import { executeOnUIRuntimeSync, runOnUI } from './threads';
 import { valueSetter } from './valueSetter';
 
 const SHOULD_BE_USE_WEB = shouldBeUseWeb();
 
+const uiValueGetter = executeOnUIRuntimeSync(<T>(sv: Mutable<T>): T => {
+  'worklet';
+  return sv.value;
+});
```

Comment:
As far as I know, this function is called only once. Also, it's name `uiValueGetter` sounds similar to `valueSetter` which has a different semantics. I think it would be better to call `executeOnUIRuntimeSync(...)()` directly inside `get value()`, the same way as we call `runOnUI(...)()` inside `set value(newValue)`.